templates/taxi/car_detail.html

```diff
@@ -1,22 +1,30 @@
 {% extends "base.html" %}
 
 {% block content %}
+  <h1>{{ car.model }}</h1>
+  <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
   <h1>
-    {{ car.model }}
-    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
-      Delete
-    </a>
-
-    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
-      Update
-    </a>
+    Drivers
+    {% if user in car.drivers.all %}
```

Comment:
you use here many to many relationship but did you fetched it from db? because otherwise here will be N+1 issue